packages/gamut-styles/src/system/types.ts

```diff
@@ -0,0 +1,124 @@
+import { SerializedStyles } from '@emotion/core';
+import { Styles } from 'polished/lib/types/style';
+import { PropAlias } from './constants';
+
+/** System Configuration */
+export type MediaQueryArray<T> =
+  | [T | undefined]
+  | [T | undefined, T | undefined]
+  | [T | undefined, T | undefined, T | undefined]
+  | [T | undefined, T | undefined, T | undefined, T | undefined]
+  | [T | undefined, T | undefined, T | undefined, T | undefined, T | undefined];
```

Comment:
I think this could be simplified to the following:
```suggestion
export type MediaQueryArray<T> = [T?, T?, T?, T?, T?];
```

Reply:
Amazing, thank you :)